### ISSUE TITLE
Label not showing in iPhone X/XR/XS

### ISSUE THREAD

**Issue by yjacobowitz** [timomeh/react-native-material-bottom-navigation#119]
### What kind of Issue is this?

Bug Report

### Actual behavior

Label not showing in FullTab component in iPhone X + XR + XS

### Expected behavior

See the label under the icon

### Additional resources

XR simulator: 
![image](https://user-images.githubusercontent.com/17860133/52273551-e0f94380-2952-11e9-844f-f4555508fb55.png)

8 Plus simulator:
![image](https://user-images.githubusercontent.com/17860133/52273575-ef475f80-2952-11e9-926f-3b4e2f987a87.png)

### Environment

| software                                | version
| --------------------------------------- | ------- 
| react-native-material-bottom-navigation | 1.0.2
| react-native or expo                    | 0.58.3
| node                                    | 10.13.0
| npm or yarn                             |6.4.1

**Comment by timomeh** [comment]
Please provide a reproductible [expo snack](https://snack.expo.io/) so I can check this.

**Comment by yjacobowitz** [comment]
https://snack.expo.io/Hko4mMD44

example without icons...I couldn't find a simulator for iphone XR in the link that you sent

**Comment by timomeh** [comment]
Thanks, but I can't reproduce your error. I ran your expo snack inside an iPhone XR Simulator and the label is visible, see this screenshot:

![image](https://user-images.githubusercontent.com/4227520/52298242-db1b5680-2982-11e9-89f5-fdb2cdf3f2dc.png)

I also extended your expo snack to use an icon, and both the icon and the label is visible: https://snack.expo.io/@timomeh/d29ya2

![image](https://user-images.githubusercontent.com/4227520/52298372-1d449800-2983-11e9-8fc2-73e48ee81ff3.png)

But I noticed that you specified a `height` for the BottomNavigation:

```js
<BottomNavigation
  style={{
    height: 56,
  // ...
```

This will most likely be your problem. The BottomNavigation on iPhone X/XR/XS get's some additional paddingBottom, because of the SafeArea of those devices. If you specify the height, this paddingBottom will cut off the bottom, and it looks like the labels aren't rendered.

I would advise you not to override the height.
